tests/ttiex355p.py

```diff
@@ -65,25 +73,25 @@ def test_WHEN_field_set_THEN_current_setpoint_updates(self, _, field):
         self._turn_on_in_const_current_mode()
 
         self.ca.set_pv_value("FIELD:SP", field)
-        self.ca.assert_that_pv_is_number("CURRENT:SP", field / AMPSTOGAUSS, tolerance=0.01)
+        self.ca.assert_that_pv_is_number("CURRENT:SP", field / AMPSTOGAUSS, tolerance=0.08)
```

Comment:
Can you move the tolerance out to a constant as it seems to be used in multiple places? Also, why have we modified it?

Reply:
Now moved out to constant. The tolerance has changed because the AMPSTOGAUSS value has changed and division in the IOC is less accurate at this number of decimal places. I have changed the AMPSTOGAUSS value because it more closely represents what is used on instruments